.github/workflows/release.yml

```diff
@@ -0,0 +1,35 @@
+name: django-allow-cidr PyPI release
+on:
+  push:
+    tags:
+      - "[0-9]+.[0-9]+.[0-9]+"
+jobs:
+  pre_release_checks:
+    runs-on: ubuntu-latest
+    name: django-allow-cidr Pre-Release CI checks
+    steps:
+      - name: Run CI checks again before publishing package
+        uses: ./.github/workflows/ci
+
+  release:
```

Comment:
What I'm seeing online is that we could move this job to the `ci` workflow file, and use some `on:` rules to ensure that it only runs when the tag is present. That way we wouldn't need to check the other workflow call and it would only run if the tests passed.

Reply:
Have updated using a step output as a sentinel to say whether or not we're in release mode. Whaddya think? Happy to keep iterating

Reply:
Hmm - glitchy at the mo. Please hold ☎️

Reply:
@pmac OK - working now using the `release` event and worth another look.